testsuite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Use default azure-cli version 2.29.0 if no version is specified in build
+ARG AZURE_CLI_VERSION=2.29.0
+FROM mcr.microsoft.com/azure-cli:${AZURE_CLI_VERSION}
```

Comment:
We can make the container more lightweight by installing just the azure storage python SDK instead of entire az package. We anyways are installing python for removing secrets from the result tar file. What is the current size of the docker image?

Reply:
Python sdk gives api which we can use in python code, currently we are using bash for all the work. We can evaluate the change required for this and then take a call. Current size of docker image is 1.32 gb

Reply:
I guess we can consider this optimization later as well, based on customer feedback